examples/albert/run_trainer.py

```diff
@@ -221,8 +191,6 @@ def main():
     training_args, dataset_args, collaboration_args, averager_args, tracker_args = parser.parse_args_into_dataclasses()
 
     logger.info(f"Found {len(collaboration_args.initial_peers)} initial peers: {collaboration_args.initial_peers}")
-    if len(collaboration_args.initial_peers) == 0:
-        raise ValueError("Please specify at least one network endpoint in initial peers.")
```

Comment:
This makes it possible to run a couple of trainers without the monitor (convenient for testing).